datasets/search.md

```diff
@@ -0,0 +1,126 @@
+# Search Data
+
+## Introduction
+
+This article introduces the datasets we maintain for search analyses.
+After reading this article,
+you should understand the search datasets well enough to produce moderately complex analyses.
+
```

Comment:
I wonder if we should make it clear at a high-level what sorts of things these datasets represent (basically counts of various things filtered on a bunch of different dimensions) and maybe call out (since it is kind of a sensitive subject) that we explicitly don't either gather or aggregate any kind of user-identifiable data (i.e. what people search for)

Reply:
I've added some additional context to the Standard Search Aggregates section. I'd like to avoid making promises here about what data we gather. That would be better kept with the data policy docs.